Better error message for: No module named build.__main__; 'build' is a package and cannot be directly executed

```
$ PKG_DIR=/home/user/code/securedrop-client make build-wheels

[...] 
Building SQLAlchemy-1.3.3
/usr/bin/python3: No module named build.__main__; 'build' is a package and cannot be directly executed
Traceback (most recent call last):
  File "/home/user/code/securedrop-debian-packaging/./scripts/build-sync-wheels", line 147, in <module>
    main()
  File "/home/user/code/securedrop-debian-packaging/./scripts/build-sync-wheels", line 118, in main
    subprocess.check_call(cmd)
  File "/usr/lib/python3.9/subprocess.py", line 373, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['python3', '-m', 'build', '--wheel', '/tmp/pip-wheel-build/SQLAlchemy-1.3.3', '--no-isolation', '-o', '/tmp/tmpxm7l_g4u']' returned non-zero exit status 1.
make: *** [Makefile:51: build-wheels] Error 1
```

This happens when you don't activate the virtualenv. We should have a better error message in this case.

--- a/scripts/build-sync-wheels
+++ b/scripts/build-sync-wheels
@@ -16,6 +16,13 @@ if os.geteuid() == 0:
     # tar has issues when resetting permissions and ends up
     # causing very subtle reproducibility issues.
     print("This script cannot be run as root.", file=sys.stderr)
+    # sys.exit(1)
+if "VIRTUAL_ENV" not in os.environ:
+    print(
+        "This script should be run in a virtualenv: "
+        "`source .venv/bin/activate`",
+        file=sys.stderr
+    )
     sys.exit(1)
 
 # Set SOURCE_DATE_EPOCH to a predictable value. Using the first

